tests/rule_based_profiler/data_assistant/test_column_value_missing_data_assistant.py

```diff
@@ -11,6 +19,109 @@
 )
 
 
+@pytest.fixture
+def sqlite_datasource_name() -> str:
+    return "sqlite_datasource"
+
+
+@pytest.fixture
+def sqlite_database_path() -> pathlib.Path:
+    relative_path = pathlib.Path(
+        "..",
+        "..",
+        "test_sets",
+        "taxi_yellow_tripdata_samples",
+        "sqlite",
+        "yellow_tripdata.db",
+    )
+    return pathlib.Path(__file__).parent.joinpath(relative_path).resolve(strict=True)
+
+
+@pytest.fixture
+def sqlite_datasource(
+    empty_data_context, sqlite_database_path, sqlite_datasource_name
+) -> SqliteDatasource:
+    connection_string = f"sqlite:///{sqlite_database_path}"
+    datasource = SqliteDatasource(
```

Comment:
@TrangPham It might be more "GX-FDS-like" to say `datasource = context.sources.add_or_create<...>(name, [...])`.  Thank you.